pages/linux/caffeinate.md

```diff
@@ -0,0 +1,16 @@
+# caffeinate
+
+> Prevent desktop from sleeping.
+> More information: <https://manned.org/caffeinate>.
+
+- Prevent from sleeping for 1 hour (3600 seconds):
+
+`caffeinate -u -t {{3600}}`
```

Comment:
Wait! this program doesn't support this example:
```bash
$ caffeinate 
usage: caffeinate [-h] [-V] COMMAND [ARGUMENT ...]
caffeinate: error: the following arguments are required: COMMAND
$ caffeinate --help
usage: caffeinate [-h] [-V] COMMAND [ARGUMENT ...]

Inhibit desktop idleness for the duration of COMMAND

positional arguments:
  COMMAND        command to run
  ARGUMENT       arguments to COMMAND

options:
  -h, --help     show this help message and exit
  -V, --version  show program's version number and exit
```
Tested on Ubuntu Jammy 22.04

Reply:
This is true, I forgot to delete this part when I copied the template from the other page apparently, sorry.